Gulpfile.js

```diff
@@ -216,3 +216,9 @@ gulp.task('test-client', ['build'], function () {
 });
 
 gulp.task('test', ['lint', 'test-server']);
+
+gulp.task('error-message-lister', function () {
```

Comment:
It's not quite good name for this actually, since it shows reports and not only errors